polkadot/xcm/src/v5/traits.rs

```diff
@@ -459,6 +459,10 @@ pub trait SendXcm {
 
 	/// Actually carry out the delivery operation for a previously validated message sending.
 	fn deliver(ticket: Self::Ticket) -> result::Result<XcmHash, SendError>;
+
+	/// Ensure `[Self::delivery]` is successful for the given `location` when called in benchmarks.
+	#[cfg(feature = "runtime-benchmarks")]
+	fn ensure_successful_delivery(location: Option<Location>);
```

Comment:
What about adding a default implementation here?  
This would prevent compilation failures (unless we intend to enforce its implementation for every `impl SendXcm`) when someone runs a command with `--feature runtime-benchmarks`, for example:
```
error[E0046]: not all trait items implemented, missing: `ensure_successful_delivery`
   --> polkadot/runtime/parachains/src/mock.rs:446:1
    |
446 | impl SendXcm for DummyXcmSender {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `ensure_successful_delivery` in implementation
    |
    = help: implement the missing item: `fn ensure_successful_delivery(_: std::option::Option<staging_xcm::v5::Location>) { todo!() }`

For more information about this error, try `rustc --explain E0046`.
warning: `polkadot-runtime-parachains` (lib test) generated 1 warning
error: could not compile `polkadot-runtime-parachains` (lib test) due to 1 previous error; 1 warning emitted

```

```suggestion
	fn ensure_successful_delivery(_location: Option<Location>) {
		()
	}
```